src/learnings/import-learnings-command.ts

```
@@ -0,0 +1,238 @@
+// CLI handler for `ivan import-learnings <csv-file>`.
+// Imports learnings from a CSV export into the canonical `.ivan/lessons.jsonl` file
+// and rebuilds the derived SQLite database. Learning IDs are deterministic hashes of
+// the statement text, so re-running the command on the same file is a no-op.
+
+import fs from 'fs';
+import path from 'path';
+import chalk from 'chalk';
```

ESLint

[error] "chalk" should be replaced with native functionality. You can instead use util.styleText. Read more here: https://e18e.dev/docs/replacements/chalk

(e18e/ban-dependencies)